site/content/en/docs/Guides/Client SDKs/_index.md

```diff
@@ -153,7 +153,9 @@ and the {{< ghlink href="examples" >}}examples{{< /ghlink >}}.
 ### Allocate()
 
 With some matchmakers and game matching strategies, it can be important for game servers to mark themselves as `Allocated`.
-For those scenarios, this SDK functionality exists. 
+For those scenarios, this SDK functionality exists.
+
+Calling `Allocate()` function does not guarantee that GameServer becomes `Allocated` after this call. For instance, it could be moved to the `Shutdown` state elsewhere (for example when fleet scales down). You can verify the result of this call by waiting for the `Allocated` Status in GameServer() function output.
```

Comment:
I would actually advocate using `Watch()` to react to when the Allocation actually happens, rather than using `GameServer()` (polling the GameServer doesn't hurt, since it's cached locally, but Watch would be better)